Blish HUD/GameServices/Input/Keyboard/KeyboardHandler.cs

```diff
@@ -242,7 +242,7 @@ public class KeyboardHandler : IInputHandler {
             // Handle text input
             if (_textInputDelegate != null) {
                 string chars = TypedInputUtil.VkCodeToString((uint)key, eventType == KeyboardEventType.KeyDown);
-                _textInputDelegate?.BeginInvoke(chars, EndTextInputAsyncInvoke, null);
+                _textInputDelegate?.Invoke(chars);
```

Comment:
We can probably remove EndTextInputAsyncInvoke with this now swapped, yeah?

Reply:
Yeah - I think I managed to undo removing it when I was cleaning up my local changes ready for the PR, I'll update to remove it.